fun will_throw
{
 try {
  [ will_throw()]
}
catch const e {
}

try { fun will_throw
{
void
try {
! 1[ will_throw()]
}
catch const e {
}return 3
}
var a   = void
var exc = void
try {[ will_throw()... ]
}
catch const e {
 e
}
assert exc
1[ will_throw()]
}
catch const e {
}return 13
}
var exc = void
try {[ will_throw()... ]
}
catch const e {
 e
}
assert exc
